### ISSUE TITLE
during drag'n'drop it is not possible to see where things land

### ISSUE THREAD

**Issue by maxandersen** [openshiftio/openshift.io#3002]
when using drag'n'drop to reorder there are no indication of where elements land.
Makes it really hard to be sure the move I'm doing is correct. 

Also when dropping things to the bottom in a list they don't reorder, i.e. the drop don't take effect.

![2018-04-11_13-54-37](https://user-images.githubusercontent.com/54129/38615199-1e4b2884-3d90-11e8-8fa3-dcdefb249c0f.gif)

**Comment by sunilmalagi** [comment]
@maxandersen @nimishamukherjee as of now its implemented as per PF-ng design, earlier I had made a suggestion, here it is: https://redhat.invisionapp.com/share/J4AKB1LCS#/220227075_01_ListView.  I can provide the color specifications if you are able to modify the same.

**Comment by catrobson** [comment]
@sunilmalagi I don't think that this code is from PF-ng, we handle drag and drop differently in that implementation:  https://rawgit.com/patternfly/patternfly-ng/master-dist/dist-demo/#/table.  It still doesn't follow recommended guidelines, so we could fix it there, but it still won't fix it in the planner since we are not pulling from patternfly-ng yet.

**Comment by sahil143** [comment]
@catrobson @sunilmalagi [ng-drag-drop](https://github.com/ObaidUrRehman/ng-drag-drop) is being used for drag and drop.
"Green plus sign" is shown under mouse cursor if the workitem is droppable. We acknowledge the problem. The implementation is little tricky. As of now, it's not blocking the drag n drop or reordering feature. We shall start working on it shortly.

**Comment by maxandersen** [comment]
the drag'n'drop on https://rawgit.com/patternfly/patternfly-ng/master-dist/dist-demo/#/table is better as rows moves "out of the way" to see where things will land.

**Comment by sunilmalagi** [comment]
@maxandersen @sahil143 @catrobson https://rawgit.com/patternfly/patternfly-ng/master-dist/dist-demo/#/table definitely looks better.. it gives clear indication of the existing and new position of the work item in the table...

**Comment by muruGanesan** [comment]
@sunilmalagi, @sahil143 
Have you done any change or update to this issue?
If yes, could you please share the solution details... so that we can close this issue.

@nimishamukherjee, @Veethika 
Is tree-view still valid? 
Are we considering 'drag & drop' in the new list view?

**Comment by Veethika** [comment]
@muruGanesan as of now we're not getting rid of the tree view completely. Drag-n-drop would still be valid for the flat list browsing.
@sunilmalagi has provided the link for the visual treatment here.

**Comment by muruGanesan** [comment]
@nimishamukherjee, @michaelkleinhenz, @sahil143 
Sunil has already provided the solution (as shown below)
https://redhat.invisionapp.com/share/J4AKB1LCS#/220227075_01_ListView. 

If it is implemented, please close this issue.

CC: @Veethika @sunilmalagi, @serenamarie125

**Comment by muruGanesan** [comment]
@joshuawilson, yes, please. We can keep this issue open. 
Since the UX team provided the recommendation, can we remove "Team/UX"?
